incognia.go

```diff
@@ -200,21 +208,31 @@ func (c *Client) RegisterFeedback(feedbackEvent FeedbackType, timestamp *time.Ti
 }
 
 func (c *Client) registerFeedback(feedbackEvent FeedbackType, timestamp *time.Time, feedbackIdentifiers *FeedbackIdentifiers) (err error) {
-	requestBody, err := json.Marshal(postFeedbackRequestBody{
-		Event:          feedbackEvent,
-		Timestamp:      timestamp.UnixNano() / 1000000,
-		InstallationID: feedbackIdentifiers.InstallationID,
-		LoginID:        feedbackIdentifiers.LoginID,
-		PaymentID:      feedbackIdentifiers.PaymentID,
-		SignupID:       feedbackIdentifiers.SignupID,
-		AccountID:      feedbackIdentifiers.AccountID,
-		ExternalID:     feedbackIdentifiers.ExternalID,
-	})
+	if !isValidFeedbackType(feedbackEvent) {
+		return ErrInvalidFeedbackType
+	}
```

Comment:
Should we validate in the library or let the backend do it?

Reply:
I think we should validate